test/intset.jl

```diff
@@ -139,6 +139,12 @@ end
 
 @testset "pop!, delete!" begin
     s = IntSet(1:2:10)
+    # deleting non-negative values should be no-op
```

Comment:
Yes fixed thanks. I updated with a CI-skipped commit -- the original one was all green except for 1 timeout.